web-jehlomat/src/screens/SetNewPassword/Components/ButtonsContainer.tsx

```diff
@@ -0,0 +1,5 @@
+﻿import {styled} from "@mui/material";
+
+export const ButtonsContainer = styled('div')({
+  textAlign: "center",
+});
```

Comment:
no new line

Reply:
Should be handle Prettier, shouldn't it?